### ISSUE TITLE
Old protobuf install in Dockerfile build

### ISSUE THREAD

**Issue by jishminor** [triton-inference-server/model_analyzer#185]
If building the model-analyzer from the provided Dockerfile, protobuf version `3.6.1` is installed leading to a failure when running the profile command:

```Traceback (most recent call last):
  File "/usr/local/bin/model-analyzer", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/model_analyzer/entrypoint.py", line 280, in main
    analyzer.cb_search(client=client)
  File "/usr/local/lib/python3.8/dist-packages/model_analyzer/analyzer.py", line 166, in cb_search
    self._model_manager.cb_search_models(self._config.profile_models)
  File "/usr/local/lib/python3.8/dist-packages/model_analyzer/model_manager.py", line 113, in cb_search_models
    self._execute_vw_search(self._config.iterations)
  File "/usr/local/lib/python3.8/dist-packages/model_analyzer/model_manager.py", line 296, in _execute_vw_search
    if not self._create_and_load_model_variant(
  File "/usr/local/lib/python3.8/dist-packages/model_analyzer/model_manager.py", line 338, in _create_and_load_model_variant
    variant_config.write_config_to_file(new_model_dir,
  File "/usr/local/lib/python3.8/dist-packages/model_analyzer/triton/model/model_config.py", line 182, in write_config_to_file
    model_config_bytes = text_format.MessageToBytes(self._model_config)
AttributeError: module 'google.protobuf.text_format' has no attribute 'MessageToBytes'
```

In the pre-built image `nvcr.io/nvidia/tritonserver:21.06-py3-sdk`, protobuf version `3.17` is installed which does have an implementation for `MessageToBytes`.

**Comment by jishminor** [comment]
A quick fix is just to do a: `python3 -m pip install --upgrade protobuf`. I'm not sure of what package is responsible for installing the old version of protobuf at this point.

**Comment by Tabrizian** [comment]
Are you on the 21.06 branch? This issue existed in the earlier versions of MA but it is fixed in the 21.06 release. I just tried building the docker from source but was not able to reproduce this issue: 

```
 model_analyzer git:(r21.06) docker run -ti  model-analyzer /bin/bash
=============================
=== Triton Model Analyzer ===
=============================
NVIDIA Release 21.06 (build 24198912)
Copyright (c) 2020-2021, NVIDIA CORPORATION.  All rights reserved.
Various files include modifications (c) NVIDIA CORPORATION.  All rights reserved.
This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license
find: File system loop detected; '/usr/bin/X11' is part of the same file system loop as '/usr/bin'.

WARNING: The NVIDIA Driver was not detected.  GPU functionality will not be available.
   Use Docker with NVIDIA Container Toolkit to start this container; see
   https://github.com/NVIDIA/nvidia-docker.

root@de60997f74a8:/opt/triton-model-analyzer# python3
Python 3.8.10 (default, Jun  2 2021, 10:49:15)
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import google.protobuf
>>> print(google.protobuf.__version__)
3.17.3
```

**Comment by jishminor** [comment]
This is on the main branch as of today. I found that the protobuf is being installed via apt via the line `RUN apt-get install -y wkhtmltopdf`. In the docker build log you can see:
```Setting up python3-protobuf (3.6.1.3-2ubuntu5)```
